client/src/components/WorkSpaceCard/index.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import LocationOnIcon from "@material-ui/icons/LocationOn";
+import PeopleIcon from "@material-ui/icons/People";
+import BusinessIcon from "@material-ui/icons/Business";
+
+import style from './WorkSpace.module.css';
+
+export const WorkSpaceCard = (props) => {
+  const [Carousel, WorkSpaceContent, Amenities, Explore, ...restProps] = props.children;
+  return (
+    <div className={style.workspace} style={props.style}>
+      {Carousel}
+      <div className={style.workspacecontent}>
+        {WorkSpaceContent}
+        {Amenities}
+        {Explore}
+        {restProps}
+      </div>
+    </div>
+  );
+}
+
+export const WorkSpaceContent = props => {
+  return (
+    <>
```

Comment:
I think we should have a wrapper div around this content.

Reply:
Not required, it will just add an extra node for no reasons